webpack/assets/javascripts/react_app/components/common/table/components/Table.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Table as PfTable } from 'patternfly-react';
+import TableBody from './TableBody';
+
+const Table = ({
+  columns,
+  rows,
+  bodyMessage,
+  children,
+  ...props
+}) => {
+  const body = children || [
+    <PfTable.Header key="header" />,
```

Comment:
no empty state? (or is that currently handled by the server ATM?)

Reply:
I'm in favor, but would love to see this merged too, so maybe as a follow up PR? (or once we move all of the components into one HOC?)